Import Fougères KO

## Comportement attendu

<!-- Décrire ce qu'il devrait se passer si tout allait bien -->
L'import Litteralis de Fougères se déroule sans erreur

## Comportement réel

<!-- Décrire ce qu'il se passe réellement -->
Plantage avec cette erreur

```
Warning: Undefined array key 1
make: *** [Makefile:157 : console] Erreur 
```

## Pour reproduire

Mettre `APP_LITTERALIS_ENABLED_ORGS=["fougeres"]` dans `.env.local`, configurer APP_LITTERALIS_ORG_FOUGERES_ID et APP_LITTERALIS_ORG_FOUGERES_CREDENTIALS, puis lancer:

```
make console CMD="app:litteralis:import"       
```

## Traces et captures d'écran

https://github.com/MTES-MCT/dialog/actions/runs/12884178417/job/35920109714#step:12:388

## Contexte supplémentaire

<!-- Tout élément de contexte permettant de mieux comprendre le problème. Par exemple, dans quel cadre l'avez-vous rencontré la première fois ? -->

## Pistes de résolution

<!-- Si vous avez des idées, même vagues, de comment investiguer et résoudre le problème, vous pouvez les indiquer ici. -->

C'est cette ligne qui pose problème

https://github.com/MTES-MCT/dialog/blob/b93ee9055860c8711bd421f4d9c0a40fdad55419/src/Infrastructure/Litteralis/LitteralisTransformer.php#L279-L280

--- a/src/Infrastructure/Litteralis/LitteralisRecordEnum.php
+++ b/src/Infrastructure/Litteralis/LitteralisRecordEnum.php
@@ -11,6 +11,7 @@ enum LitteralisRecordEnum: string
     case COUNT_EXTRACTED_FEATURES = 'litteralis.extracted_features';
     case COUNT_IMPORTED_FEATURES = 'litteralis.imported_features';
 
+    case ERROR_MEASURE_PARAMETER_MALFORMED = 'litteralis.measure_parameter_malformed';
     case ERROR_MEASURE_PARAMETER_INCONSISTENT_NUMBER = 'litteralis.measure_parameter_inconsistent_number';
     case ERROR_MAX_SPEED_VALUE_INVALID = 'litteralis.max_speed_value_invalid';
     case ERROR_MAX_SPEED_VALUE_MISSING = 'litteralis.max_speed_value_missing';

--- a/src/Infrastructure/Litteralis/LitteralisTransformer.php
+++ b/src/Infrastructure/Litteralis/LitteralisTransformer.php
@@ -110,8 +110,8 @@ final readonly class LitteralisTransformer
     private function setCategory(SaveRegulationGeneralInfoCommand $generalInfoCommand, array $properties): void
     {
         $generalInfoCommand->category = $properties['documenttype'] === 'ARRETE PERMANENT'
-        ? RegulationOrderCategoryEnum::PERMANENT_REGULATION->value
-        : RegulationOrderCategoryEnum::TEMPORARY_REGULATION->value;
+            ? RegulationOrderCategoryEnum::PERMANENT_REGULATION->value
+            : RegulationOrderCategoryEnum::TEMPORARY_REGULATION->value;
     }
 
     private function setSubject(SaveRegulationGeneralInfoCommand $generalInfoCommand, array $properties): void
@@ -277,7 +277,22 @@ final readonly class LitteralisTransformer
 
         foreach ($measureParameters as $param) {
             // Exemple : "Circulation interdite 2 | dérogation : urgences" -> ['Circulation interdite 2', 'dérogation : urgences']
-            [$name, $item] = $this->parseSeparatedString($param, '|');
+            $parts = explode('|', $param, 2);
+
+            if (\count($parts) !== 2) {
+                $reporter->addError(LitteralisRecordEnum::ERROR_MEASURE_PARAMETER_MALFORMED->value, [
+                    CommonRecordEnum::ATTR_REGULATION_ID->value => $properties['arretesrcid'],
+                    CommonRecordEnum::ATTR_URL->value => $properties['shorturl'],
+                    CommonRecordEnum::ATTR_DETAILS->value => [
+                        'idemprise' => $properties['idemprise'],
+                        'param' => $param,
+                    ],
+                ]);
+
+                continue;
+            }
+
+            [$name, $item] = array_map(fn ($s) => trim($s), $parts);
 
             if (preg_match('/^(?P<name>[\s|\w]+) (?P<number>\d+)$/i', $name, $matches)) {
                 // Si un numéro est indiqué, il doit correspondre au index commençant à 1 de la mesure dans le champ 'mesures'.

--- a/tests/Unit/Infrastructure/Litteralis/LitteralisTransformerTest.php
+++ b/tests/Unit/Infrastructure/Litteralis/LitteralisTransformerTest.php
@@ -250,6 +250,18 @@ final class LitteralisTransformerTest extends TestCase
             self::feature,
         );
 
+        // Trigger ERROR_MEASURE_PARAMETER_MALFORMED
+        $malformedParamFeature = str_replace(
+            '"mesures": "Limitation de vitesse"',
+            '"mesures": "Circulation interdite"',
+            self::feature,
+        );
+        $malformedParamFeature = str_replace(
+            '"parametresmesures": "Limitation de vitesse | limite de vitesse : 70 km/h"',
+            '"parametresmesures": "Circulation interdite 1"',
+            $malformedParamFeature,
+        );
+
         // Trigger ERROR_MEASURE_PARAMETER_INCONSISTENT_NUMBER
         $inconsistentIndexFeature = str_replace(
             '"mesures": "Limitation de vitesse"',
@@ -273,6 +285,7 @@ final class LitteralisTransformerTest extends TestCase
             json_decode($invalidDatesFeature, associative: true),
             json_decode($missingSpeedLimitValueFeature, associative: true),
             json_decode($invalidSpeedLimitValueFeature, associative: true),
+            json_decode($malformedParamFeature, associative: true),
             json_decode($inconsistentIndexFeature, associative: true),
             json_decode($periodUnparsableFeature, associative: true),
         ];
@@ -328,6 +341,18 @@ final class LitteralisTransformerTest extends TestCase
                     CommonRecordEnum::ATTR_DETAILS->value => [
                         'idemprise' => 493136,
                         'limite de vitesse' => 'foo km/h',
+                    ],
+                ],
+            ],
+            [
+                RecordTypeEnum::ERROR->value,
+                [
+                    RecordTypeEnum::ERROR->value => LitteralisRecordEnum::ERROR_MEASURE_PARAMETER_MALFORMED->value,
+                    CommonRecordEnum::ATTR_REGULATION_ID->value => '24-A-0126',
+                    CommonRecordEnum::ATTR_URL->value => 'https://dl.sogelink.fr/?n3omzTyS',
+                    CommonRecordEnum::ATTR_DETAILS->value => [
+                        'idemprise' => 493136,
+                        'param' => 'Circulation interdite 1',
                     ],
                 ],
             ],

--- a/translations/messages.fr.xlf
+++ b/translations/messages.fr.xlf
@@ -2491,6 +2491,10 @@
                 <source>integration.report.error.litteralis.period_unparsable</source>
                 <target>Emprises avec "jours et horaires" au format non-supporté</target>
             </trans-unit>
+            <trans-unit id="integration.report.error.litteralis.measure_parameter_malformed">
+                <source>integration.report.error.litteralis.measure_parameter_malformed</source>
+                <target>Emprises avec paramètres de mesures au format inattendu</target>
+            </trans-unit>
             <trans-unit id="integration.report.error.litteralis.measure_parameter_inconsistent_number">
                 <source>integration.report.error.litteralis.measure_parameter_inconsistent_number</source>
                 <target>Emprises avec numéros de mesures incohérents</target>

